code/model/DNProject.php

```diff
@@ -76,6 +76,14 @@ class DNProject extends DataObject {
 	protected static $relation_cache = array();
 
 	/**
+	 * Member::currentUser() cache to avoid re-query the same thing
+	 * again and again.
+	 *
+	 * @var mixed
+	 */
+	private static $_cache_member = null;
```

Comment:
can you maybe rename it to $_cache_current_member? otherwise it sounds like a generic hashed member cache